src/components/NcAppSettingsDialog/NcAppSettingsDialogVersion.vue

```diff
@@ -0,0 +1,45 @@
+<!--
+  - SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<script lang="ts">
+import { loadState } from '@nextcloud/initial-state'
+import { APP_NAME } from '../../utils/appName.ts'
+import { once } from '../../utils/utils.ts'
+
+const getLocalizedAppName = once(() => {
+	const activeApp = loadState<string>('core', 'active-app', APP_NAME)
+	return loadState<{ id: string, name: string }[]>('core', 'apps', []).find(({ id }) => id === activeApp)?.name ?? APP_NAME
+})
+</script>
+
+<script setup lang="ts">
+
+import { inject } from 'vue'
+import { APP_VERSION } from '../../utils/appName.ts'
+import { APP_SETTINGS_LEGACY_DESIGN_KEY } from './useAppSettingsDialog.ts'
+
+const legacy = inject(APP_SETTINGS_LEGACY_DESIGN_KEY)!
+
+const localizedAppName = getLocalizedAppName()
+</script>
+
+<template>
+	<div :class="[$style.appSettingsDialogVersion, { [$style.appSettingsDialogVersion__legacy]: legacy }]">
+		{{ localizedAppName }} version {{ APP_VERSION }}
```

Comment:
<img width="288" height="74" alt="Image" src="https://github.com/user-attachments/assets/a5761001-c73e-42df-9be9-0004fd52f6b3" />

Since app name is localised, I'd make the whole string translated. At least that's what I see in some other apps

Reply:
I would go with `{appName} {version}`